src/helpers/sendMessageHelpers.ts

```diff
@@ -0,0 +1,14 @@
+export function messageHandler(message: string, url: string): string {
+    const parentWindow = window.opener;
+    const sendMessage = parentWindow.postMessage(message, url);
+    return sendMessage;
+}
+
+export function closeWindowHandler() {
+    return window.close();
+}
+
+export function sendMessage(message: string) {
+    messageHandler(message, import.meta.env.VITE_HOST_SIMPLE_SIGNER_DEMO);
```

Comment:
this doesn't go here, messages always come from/go to the parent window, doesn't matter which URL